src/utils/image.ts

```diff
@@ -78,32 +78,41 @@ export const getFirebaseStorageResizedImage = (
 // @see https://crypto.stackexchange.com/questions/8533/why-are-bitwise-rotations-used-in-cryptography/8534#8534
 const DIFFUSION_PRIME = 31;
 
-type DetermineAvatar = (options?: {
+type DetermineAvatarProps = {
```

Comment:
Please reserve the Props suffix for types that actually are props of components and add Options and Result to utility functions. This is a little convention that makes it easily determine the use.